Library/Formula/ndenv.rb

```diff
@@ -0,0 +1,30 @@
+class Ndenv < Formula
+  homepage "https://github.com/riywo/ndenv"
+  url "https://github.com/riywo/ndenv/archive/v0.4.0.tar.gz"
+  sha1 "f07199b2dd4e28ef9e63e1a9cf17f71c874a1c61"
+
+  def install
+    inreplace "libexec/ndenv", "/usr/local", HOMEBREW_PREFIX
+    prefix.install Dir["*"]
+
+    #  Create a plugin and link it into ndenv's opt_prefix. Only create if it
+    #    does not already exist (user might have installed stuff manually).
+    plugins_path = Pathname.new "#{HOMEBREW_PREFIX}/share/ndenv/plugins"
+    plugins_path.mkpath
+
+    prefix.install_symlink "#{plugins_path}"
+  end
+
+  def caveats; <<-EOS.undent
+    To use Homebrew's directories rather than ~/.ndenv add to your profile:
+      export NDENV_ROOT=#{opt_prefix}
+
+    To enable shims and autocompletion add to your profile:
+      if which ndenv > /dev/null; then eval "$(ndenv init -)"; fi
+    EOS
+  end
+
+  test do
+    system "#{bin}/ndenv --version"
```

Comment:
`system "#{bin}/ndenv", "--version"` instead of `system "#{bin}/ndenv --version"`  (and same for the other test)